statefun-tasks/src/main/java/com/sbbsystems/statefun/tasks/batchcallback/messagehandlers/SimpleBatchSubmitter.java

```diff
@@ -0,0 +1,61 @@
+/*
+ * Copyright [2023] [Frans King, Luke Ashworth]
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.sbbsystems.statefun.tasks.batchcallback.messagehandlers;
+
+import com.sbbsystems.statefun.tasks.batchcallback.CallbackFunctionState;
+import com.sbbsystems.statefun.tasks.generated.ResultsBatch;
+import com.sbbsystems.statefun.tasks.types.MessageTypes;
+import org.apache.flink.statefun.sdk.Context;
+import org.apache.flink.statefun.sdk.FunctionType;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+import java.util.stream.Collectors;
+import java.util.stream.StreamSupport;
+
+public class SimpleBatchSubmitter implements BatchSubmitter {
```

Comment:
We can add other implementations in the future, e.g. a throttled version with a limited batch size